src/sentry/search/django/backend.py

```diff
@@ -344,16 +323,6 @@ def query(self, projects, tags=None, environments=None, sort_by='date', limit=10
         if tags is None:
             tags = {}
 
-        try:
```

Comment:
I figured that these don't really belong here, since we usually handle conversion outside, so moved them